README.md

```diff
@@ -1,8 +1,10 @@
-# ledger-oasis-go
+# Oasis Core Ledger
```

Comment:
```suggestion
# Oasis Core Ledger Hardware Signer
```

Reply:
Also maybe mention somewhere below what "ledger" refers to as it is a general term.

Reply:
I would prefer to keep the project's name in line with the repo's name. Also, the project's scope could be a bit wider than just providing a Ledger signer for Oasis Core.

I've tried to describe all this in the updated README. Please, take another look.

Reply:
Looks good!